apps/condo/domains/news/components/NewsForm/BaseNewsForm.tsx

```diff
@@ -127,6 +125,25 @@ const BIG_HORIZONTAL_GUTTER: [Gutter, Gutter] = [50, 0]
 const ALL_SQUARE_BRACKETS_OCCURRENCES_REGEX = /\[[^\]]*?\]/g
 const ADDITIONAL_DISABLED_MINUTES_COUNT = 5
 
+const getCounterStyle = (textLength: number): React.CSSProperties => {
```

Comment:
Maybe "build" prefix to not mix with data fetching?
```suggestion
const buildCounterStyle = (textLength: number): React.CSSProperties => {
```

Reply:
done